src/notation/internal/notationparts.cpp

```diff
@@ -1329,7 +1329,9 @@ void NotationParts::updateSoloist(const PartInstrumentList& parts)
 
 void NotationParts::sortParts(const PartInstrumentList& parts, const QList<Ms::Staff*>& originalStaves)
 {
-    Q_ASSERT(!originalStaves.empty());
+    if (originalStaves.empty()) {
+        return;
+    }
 
     Ms::Score* score = originalStaves[0]->score();
```

Comment:
@njvdberg app crashes here for some unclear reasons (after creating a new score that contains several instruments). I have added a check on emptiness, but I'm not sure if this is ok because there was an assert

Reply:
Fixed in https://github.com/musescore/MuseScore/pull/8341